apps/asap-server/src/config.ts

```diff
@@ -44,3 +44,4 @@ export const algoliaAppId = ALGOLIA_APP_ID || 'LVYWOPQ0A9';
 export const algoliaSearchApiKey = ALGOLIA_SEARCH_API_KEY || '';
 export const algoliaResearchOutputIndex =
   ALGOLIA_RESEARCH_OUTPUT_INDEX || 'asap-hub_research_outputs_dev';
+export const algoliaApiKeyTtl = 36060;
```

Comment:
casing 😅

Reply:
whats wrong with it in this var name?

Reply:
ALGOLIA_API_KEY_TTL

Reply:
meh I don't know what the convention is in the backend. Thats how I would normally expect to find a constant